2015/12/29 12:05:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'phone'' in C:\xampp\htdocs\vbapp1.com\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\vbapp1.com\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\vbapp1.com\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\vbapp1.com\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\vbapp1.com\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\vbapp1.com\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\vbapp1.com\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/gii/model/index
HTTP_REFERER=http://localhost/vbapp1.com/index.php/gii/default/index
---
2015/12/30 14:14:15 [error] [php] Undefined variable: model (C:\xampp\htdocs\vbapp1.com\themes\modern\views\layouts\main.php:55)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\widgets\CContentDecorator.php(76): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate()
#2 C:\xampp\htdocs\vbapp1.com\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CBaseController.php(214): CContentDecorator->run()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CBaseController.php(308): NumbersController->endWidget()
#5 C:\xampp\htdocs\vbapp1.com\protected\views\layouts\column2.php(22): NumbersController->endContent()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CBaseController.php(126): require()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CBaseController.php(95): NumbersController->renderInternal()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(784): NumbersController->renderFile()
#9 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(128): NumbersController->render()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionIndex()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#15 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#16 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#17 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#19 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#20 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#21 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#22 C:\xampp\htdocs\vbapp1.com\index.php(13): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/index
in C:\xampp\htdocs\vbapp1.com\themes\modern\views\layouts\main.php (55)
in C:\xampp\htdocs\vbapp1.com\protected\views\layouts\column2.php (22)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (128)
2016/01/06 16:00:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Невозможно обработать запрос "material/search".' in C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('material/search')
#1 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/vbapp1.com/index.php/material/search
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/06 16:01:52 [error] [exception.CException] exception 'CException' with message 'Контроллер NumbersController не может найти представление "found".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): CController->renderPartial('found', Array, true)
#1 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(208): CController->render('found', Array)
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#13 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/06 16:02:28 [error] [exception.CException] exception 'CException' with message 'Контроллер NumbersController не может найти представление "found".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): CController->renderPartial('found', Array, true)
#1 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(208): CController->render('found', Array)
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#13 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/06 16:02:29 [error] [exception.CException] exception 'CException' with message 'Контроллер NumbersController не может найти представление "found".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): CController->renderPartial('found', Array, true)
#1 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): CController->render('found', Array)
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#13 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/06 16:02:30 [error] [exception.CException] exception 'CException' with message 'Контроллер NumbersController не может найти представление "found".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): CController->renderPartial('found', Array, true)
#1 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): CController->render('found', Array)
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#13 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/06 16:02:31 [error] [exception.CException] exception 'CException' with message 'Контроллер NumbersController не может найти представление "found".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): CController->renderPartial('found', Array, true)
#1 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): CController->render('found', Array)
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#13 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/06 16:02:31 [error] [exception.CException] exception 'CException' with message 'Контроллер NumbersController не может найти представление "found".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): CController->renderPartial('found', Array, true)
#1 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): CController->render('found', Array)
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#13 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/06 16:02:31 [error] [exception.CException] exception 'CException' with message 'Контроллер NumbersController не может найти представление "found".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): CController->renderPartial('found', Array, true)
#1 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): CController->render('found', Array)
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#13 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\vbapp1.com\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/06 16:06:00 [error] [php] Undefined variable: phones (C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php:8)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): NumbersController->render()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(13): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search
in C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php (8)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (202)
in C:\xampp\htdocs\vbapp1.com\index.php (13)
2016/01/06 16:06:50 [error] [php] Undefined variable: phones (C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php:9)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): NumbersController->render()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(13): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search
in C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php (9)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (202)
in C:\xampp\htdocs\vbapp1.com\index.php (13)
2016/01/06 16:07:05 [error] [php] Undefined variable: phones (C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php:9)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): NumbersController->render()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(13): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search
in C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php (9)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (202)
in C:\xampp\htdocs\vbapp1.com\index.php (13)
2016/01/06 16:07:06 [error] [php] Undefined variable: phones (C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php:5)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): NumbersController->render()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(13): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search
in C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php (5)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (202)
in C:\xampp\htdocs\vbapp1.com\index.php (13)
2016/01/06 16:07:10 [error] [php] Undefined variable: phones (C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php:5)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): NumbersController->render()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(13): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search
in C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php (5)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (202)
in C:\xampp\htdocs\vbapp1.com\index.php (13)
2016/01/06 16:07:37 [error] [php] Undefined variable: phones (C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php:5)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(202): NumbersController->render()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(13): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search
in C:\xampp\htdocs\vbapp1.com\protected\views\numbers\found.php (5)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (202)
in C:\xampp\htdocs\vbapp1.com\index.php (13)
2016/01/06 16:13:04 [error] [php] include(SiteSearch.php): failed to open stream: No such file or directory (C:\xampp\htdocs\vbapp1.com\framework\YiiBase.php:432)
Stack trace:
#0 unknown(0): spl_autoload_call()
#1 C:\xampp\htdocs\vbapp1.com\framework\YiiBase.php(303): class_exists()
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWidgetFactory.php(146): import()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CBaseController.php(146): CWidgetFactory->createWidget()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CBaseController.php(180): NumbersController->createWidget()
#5 C:\xampp\htdocs\vbapp1.com\themes\modern\views\layouts\main.php(41): NumbersController->widget()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CBaseController.php(126): require()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CBaseController.php(95): NumbersController->renderInternal()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(784): NumbersController->renderFile()
#9 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(129): NumbersController->render()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(49): NumbersController->actionIndex()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#15 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#16 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#17 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#19 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#20 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#21 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#22 C:\xampp\htdocs\vbapp1.com\index.php(13): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/index
in C:\xampp\htdocs\vbapp1.com\themes\modern\views\layouts\main.php (41)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (129)
in C:\xampp\htdocs\vbapp1.com\index.php (13)
2016/01/10 16:33:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "numbers".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(270): CController->missingAction('numbers')
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('numbers')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/numbers')
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/numbers/5?text=32423523
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 16:33:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "numbers".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(270): CController->missingAction('numbers')
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('numbers')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/numbers')
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/numbers/5?text=32423523
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 16:33:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "numbers".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(270): CController->missingAction('numbers')
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('numbers')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/numbers')
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/numbers/5
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 16:34:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "numbers".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(270): CController->missingAction('numbers')
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('numbers')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/numbers')
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/numbers/5
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 16:35:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "numbers".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(270): CController->missingAction('numbers')
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('numbers')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/numbers')
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/numbers/5
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 16:35:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "numbers".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(270): CController->missingAction('numbers')
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('numbers')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/numbers')
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/numbers/6
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 16:35:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "5".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(270): CController->missingAction('5')
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('5')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/5')
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/5/5
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 16:35:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Системе не удалось найти запрашиваемое действие "5".' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(270): CController->missingAction('5')
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('5')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/5')
#3 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#5 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/5/5
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 17:41:06 [error] [php] Undefined variable: dataProvider (C:\xampp\htdocs\vbapp1.com\protected\views\numbers\index.php:18)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(187): NumbersController->render()
#3 unknown(0): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#17 C:\xampp\htdocs\vbapp1.com\index.php(15): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=1312312&yt0=submit
in C:\xampp\htdocs\vbapp1.com\protected\views\numbers\index.php (18)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (187)
in C:\xampp\htdocs\vbapp1.com\index.php (15)
2016/01/10 17:41:47 [error] [php] Undefined variable: dataProvider (C:\xampp\htdocs\vbapp1.com\protected\views\numbers\index.php:18)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(187): NumbersController->render()
#3 unknown(0): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#17 C:\xampp\htdocs\vbapp1.com\index.php(15): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=1312312&yt0=submit
in C:\xampp\htdocs\vbapp1.com\protected\views\numbers\index.php (18)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (187)
in C:\xampp\htdocs\vbapp1.com\index.php (15)
2016/01/10 18:09:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:158
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(184): NumbersController->loadModel('32423523')
#1 [internal function]: NumbersController->actionSearch('32423523')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=32423523&yt0=submit
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 18:09:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:158
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(184): NumbersController->loadModel('312312412')
#1 [internal function]: NumbersController->actionSearch('312312412')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=312312412&yt0=submit
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 18:11:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:191
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(184): NumbersController->searchByPhone('123123124')
#1 [internal function]: NumbersController->actionSearch('123123124')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=123123124&yt0=submit
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 18:11:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:191
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(184): NumbersController->searchByPhone('123123124')
#1 [internal function]: NumbersController->actionSearch('123123124')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=123123124&yt1=submit
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/13
---
2016/01/10 18:23:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Phone was not found.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:191
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(184): NumbersController->searchByPhone('')
#1 [internal function]: NumbersController->actionSearch('')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 18:23:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Phone was not found.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:194
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(185): NumbersController->searchByPhone('')
#1 [internal function]: NumbersController->actionSearch('')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 18:23:57 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Phone was not found.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:194
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(185): NumbersController->searchByPhone('')
#1 [internal function]: NumbersController->actionSearch('')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/search?search=
---
2016/01/10 18:23:58 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Phone was not found.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:194
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(185): NumbersController->searchByPhone('')
#1 [internal function]: NumbersController->actionSearch('')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/search?search=
---
2016/01/10 18:25:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Phone was not found.' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:193
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(184): NumbersController->searchByPhone('23212')
#1 [internal function]: NumbersController->actionSearch('23212')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=23212
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 18:25:25 [error] [exception.CHttpException.Phone was not found.] exception 'CHttpException' in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php:193
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(184): NumbersController->searchByPhone('213213')
#1 [internal function]: NumbersController->actionSearch('213213')
#2 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#3 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#4 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('search')
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/search')
#14 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#15 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#16 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=213213
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 18:26:14 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\vbapp1.com\themes\modern\views\numbers\view.php:8)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(185): NumbersController->render()
#3 unknown(0): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#17 C:\xampp\htdocs\vbapp1.com\index.php(15): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=s
in C:\xampp\htdocs\vbapp1.com\themes\modern\views\numbers\view.php (8)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (185)
in C:\xampp\htdocs\vbapp1.com\index.php (15)
2016/01/10 18:26:20 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\vbapp1.com\themes\modern\views\numbers\view.php:8)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(185): NumbersController->render()
#3 unknown(0): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#17 C:\xampp\htdocs\vbapp1.com\index.php(15): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=2312
in C:\xampp\htdocs\vbapp1.com\themes\modern\views\numbers\view.php (8)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (185)
in C:\xampp\htdocs\vbapp1.com\index.php (15)
2016/01/10 18:26:24 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\vbapp1.com\themes\modern\views\numbers\view.php:8)
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(869): NumbersController->renderFile()
#1 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(782): NumbersController->renderPartial()
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(185): NumbersController->render()
#3 unknown(0): NumbersController->actionSearch()
#4 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): NumbersController->runAction()
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): NumbersController->filterAccessControl()
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): NumbersController->runActionWithFilters()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): NumbersController->run()
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#17 C:\xampp\htdocs\vbapp1.com\index.php(15): CWebApplication->run()
REQUEST_URI=/vbapp1.com/index.php/numbers/search?search=321%5C
in C:\xampp\htdocs\vbapp1.com\themes\modern\views\numbers\view.php (8)
in C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php (185)
in C:\xampp\htdocs\vbapp1.com\index.php (15)
2016/01/10 19:01:46 [error] [exception.CDbException] exception 'CDbException' with message 'Имя столбца должно быть строкой или массивом.' in C:\xampp\htdocs\vbapp1.com\framework\db\schema\CDbCommandBuilder.php:831
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\db\schema\CDbCommandBuilder.php(624): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 C:\xampp\htdocs\vbapp1.com\framework\db\ar\CActiveRecord.php(1490): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(155): CActiveRecord->findByPk('1')
#3 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(56): NumbersController->loadModel('1')
#4 [internal function]: NumbersController->actionView('1')
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#6 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('view')
#16 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/view')
#17 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#18 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#19 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/1
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/index
---
2016/01/10 19:01:59 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Некорректный запрос.' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/update')
#10 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#12 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/update/
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/admin
---
2016/01/10 19:02:04 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Некорректный запрос.' in C:\xampp\htdocs\vbapp1.com\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/update')
#10 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#11 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#12 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/update/
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/admin
---
2016/01/10 19:02:19 [error] [exception.CDbException] exception 'CDbException' with message 'Имя столбца должно быть строкой или массивом.' in C:\xampp\htdocs\vbapp1.com\framework\db\schema\CDbCommandBuilder.php:831
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\db\schema\CDbCommandBuilder.php(624): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 C:\xampp\htdocs\vbapp1.com\framework\db\ar\CActiveRecord.php(1490): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(155): CActiveRecord->findByPk('1')
#3 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(56): NumbersController->loadModel('1')
#4 [internal function]: NumbersController->actionView('1')
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#6 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('view')
#16 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/view')
#17 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#18 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#19 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/1
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/
---
2016/01/10 19:03:12 [error] [exception.CDbException] exception 'CDbException' with message 'Имя столбца должно быть строкой или массивом.' in C:\xampp\htdocs\vbapp1.com\framework\db\schema\CDbCommandBuilder.php:831
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\db\schema\CDbCommandBuilder.php(624): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 C:\xampp\htdocs\vbapp1.com\framework\db\ar\CActiveRecord.php(1490): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '0', '', Array, '`t`.')
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(155): CActiveRecord->findByPk('0')
#3 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(56): NumbersController->loadModel('0')
#4 [internal function]: NumbersController->actionView('0')
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#6 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('view')
#16 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/view')
#17 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#18 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#19 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/0
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/
---
2016/01/10 19:03:54 [error] [exception.CDbException] exception 'CDbException' with message 'Имя столбца должно быть строкой или массивом.' in C:\xampp\htdocs\vbapp1.com\framework\db\schema\CDbCommandBuilder.php:831
Stack trace:
#0 C:\xampp\htdocs\vbapp1.com\framework\db\schema\CDbCommandBuilder.php(624): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 C:\xampp\htdocs\vbapp1.com\framework\db\ar\CActiveRecord.php(1490): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(155): CActiveRecord->findByPk('1')
#3 C:\xampp\htdocs\vbapp1.com\protected\controllers\NumbersController.php(56): NumbersController->loadModel('1')
#4 [internal function]: NumbersController->actionView('1')
#5 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(NumbersController), Array)
#6 C:\xampp\htdocs\vbapp1.com\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(NumbersController), Object(ReflectionMethod), Array)
#7 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilter.php(40): CFilterChain->run()
#10 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\vbapp1.com\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(291): CFilterChain->run()
#14 C:\xampp\htdocs\vbapp1.com\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(282): CController->run('view')
#16 C:\xampp\htdocs\vbapp1.com\framework\web\CWebApplication.php(141): CWebApplication->runController('numbers/view')
#17 C:\xampp\htdocs\vbapp1.com\framework\base\CApplication.php(184): CWebApplication->processRequest()
#18 C:\xampp\htdocs\vbapp1.com\index.php(15): CApplication->run()
#19 {main}
REQUEST_URI=/vbapp1.com/index.php/numbers/1
HTTP_REFERER=http://localhost/vbapp1.com/index.php/numbers/
---
